src/Http/ApiClient.php

```diff
@@ -188,7 +188,7 @@ private function initializeHttpClient()
                     self::OPTION_HANDLER => $middleware,
                     self::OPTION_VERIFY => true,
                     self::OPTION_CURL => [
-                        CURLOPT_PINNEDPUBLICKEY => $this->determinePinnedServerPublicKey(),
+                        //CURLOPT_PINNEDPUBLICKEY => $this->determinePinnedServerPublicKey(),
```

Comment:
Wut? Why is it commented out? :D

Reply:
Shit yea haha, m y curl doesn’t work neither and didn’t fix it 😁. Therefore I must comment this line out for the sdk to work. Obviously forgot to revert it.